chainstate/src/detail/chainstateref/mod.rs

```diff
@@ -63,7 +63,7 @@ pub struct ChainstateRef<'a, S, V> {
     chainstate_config: &'a ChainstateConfig,
     tx_verification_strategy: &'a V,
     db_tx: S,
-    time_getter: &'a TimeGetterFn,
+    time_getter: &'a dyn TimeGetterFn,
```

Comment:
I think the right thing to put here is `TimeGetter`

Reply:
OK, updated.